### PR TITLE
CHANGELOG: Add entry for etcdutl hashkv command.

### PR DESCRIPTION
**Pull request by mello369**
Issue : https://github.com/etcd-io/etcd/issues/15061

### CODE REVIEW

File: CHANGELOG/CHANGELOG-3.6.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ See [code changes](https://github.com/etcd-io/etcd/compare/v3.5.0...v3.6.0).
 - Add `migrate` command for downgrading/upgrading etcd data dir files.
 - Add [optional --bump-revision and --mark-compacted flag to etcdutl snapshot restore operation](https://github.com/etcd-io/etcd/pull/16029).
 - Removed [legacy etcdutl backup](https://github.com/etcd-io/etcd/pull/16662)
+- Add [hashkv](https://github.com/etcd-io/etcd/pull/15965) command to print hash of keys and values up to given revision
```
Comment (jmhbnz) [inline CHANGELOG/CHANGELOG-3.6.md:44]: Nitpick: Suggest moving the line up one so all the `Add` entries are together.

**Review by jmhbnz — COMMENTED** [review]
Thanks for taking care of this @mello369!  Just one minor suggestion below if you have time 🙏🏻 

/ok-to-test

**Comment by mello369** [comment]
> Thanks for taking care of this @mello369! Just one minor suggestion below if you have time 🙏🏻
> 
> /ok-to-test

Thanks for the suggestion @jmhbnz. Updated!

**Review by jmhbnz — APPROVED** [review]
LGTM - Thanks @mello369

**Comment by mello369** [comment]
@jmhbnz forgive me for the naive question, but do I need other approvals on this PR?

**Review by ahrtr — APPROVED** [review]

**Comment by jmhbnz** [comment]
> @jmhbnz forgive me for the naive question, but do I need other approvals on this PR?

Normally we leave things open a bit for second pair of eyes in case a reviewer misses something.  Thanks @ahrtr for additional review, merging now.

**Pull request merged**